### DISCUSSION TITLE
YOLO V8 Implementation

### DISCUSSION THREAD

**Discussion by DanielVarelaMenendez** [cgtinker/BlendArMocap#156] [Ideas]
First of all thank you, this project is amazing!
I am currently working into implement new YOLO V8 with FreeMocap.
I wonder if anyone already working on this. I would not like to duplicate efforts 😄

**Comment by cgtinker** [comment]
Hey, you should ask that at the [freemocap project](https://github.com/freemocap/freemocap).
IDK how to make use of YOLOV8 data in blender but if you have ideas let me know =)
Seems cool for sure!
